XmlFormatterOsIndependent/ViewModels/SettingsWindowViewModel.cs

```diff
@@ -0,0 +1,197 @@
+﻿using PluginFramework.DataContainer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Analyzer 'SonarAnalyzer.Rules.CSharp.ConditionEvaluatesToConstant' threw an exception of type 'SonarAnalyzer.SymbolicExecution.SymbolicExecutionException' with message 'Error processing method: GetSettingsValue ## Method file: SettingsWindowViewModel.cs ## Method line: 158,8 ## Inner exception: System.NullReferenceException: Object reference not set to an instance of an object ##   at SonarAnalyzer.SymbolicExecution.CSharpExplodedGraph.VisitDefaultExpression (Microsoft.CodeAnalysis.SyntaxNode instruction, S](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)